docs/user_guide/quickstart.rst

```diff
@@ -256,8 +256,12 @@ of popular news source urls.. In case you need help choosing a news source!
       it              Italian
       ko              Korean
       no              Norwegian
+      fa              Persian
       pl              Polish
       pt              Portuguese
       sv              Swedish
       zh              Chinese
       uk              Ukrainian
+      sw              Swahili
+
```

Comment:
Extra newlines, but I'll still approve we can remove later

Reply:
yea sorry about that, the travis 3.4 build randomly failed and I had to to do something to re-trigger